config/backend-config/backend-config.go

```diff
@@ -133,9 +133,10 @@ type SourceRegulationT struct {
 }
 
 type SourcesT struct {
-	EnableMetrics bool      `json:"enableMetrics"`
-	WorkspaceID   string    `json:"workspaceId"`
-	Sources       []SourceT `json:"sources"`
+	EnableMetrics bool       `json:"enableMetrics"`
+	WorkspaceID   string     `json:"workspaceId"`
+	Sources       []SourceT  `json:"sources"`
+	Libraries     LibrariesT `json:"libraries"`
```

Comment:
rename these to LibraryIDs as we are getting only the library ID in the source right?